config/config.yml

```diff
@@ -167,6 +167,12 @@ TAGS_PER_SEARCH_PAGE: 50
 # When updating tag counts, how many to do in one transaction
 TAG_UPDATE_BATCH_SIZE: 100
 
+# We only start caching tag counts for tags used more than a certain number of times
+TAGGINGS_COUNT_MIN_CACHE_COUNT: 1000
+
+# For tagging changes, we only reindex tags used less than a certain number of times
+TAGGINGS_COUNT_REINDEX_LIMIT: 1000
```

Comment:
I think this default can be a fair amount lower, since `suggested_parent_tags` only checks the first 50 works by default, and I suspect that there aren't all that many tags with 50+ drafts that don't have any posted works. But it's just a default so it's easy enough to change later, too.